modules/krona/ktimporttext/main.nf

```diff
@@ -0,0 +1,43 @@
+process KRONA_KTIMPORTTEXT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::krona=2.8.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/krona:2.8.1--pl5321hdfd78af_1':
+        'quay.io/biocontainers/krona:2.8.1--pl5321hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(report)
+
+    output:
+    tuple val(meta), path ('*.html'), emit: html
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    fs=($report)
+
+    if [ "\${#fs[@]}" -eq "1" ]; then
+        ktImportText  \\
+            $args \\
+            -o ${prefix}.html \\
+            $report,${meta.id}
```

Comment:
Is this really necessary? Why not always just have the base name of the file - for consistency? By default ${prefix} is already `${meta.id}` so for any input files, this would basically be the same as the basename that ktImporText woudl be generating anyway?